### PR TITLE
Gateway - Update (or discord.py Update) » intents

### PR DESCRIPTION
**Pull request by CnyAnime**
This change is for the newest version of discord.py (gateway change)
image

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -25,6 +25,9 @@
 from utils.caches import cache, CacheManager
 from logging.handlers import RotatingFileHandler
 
+# this section is for the new gateway (latest discord.py version)
```
Comment (CnyAnime) [inline bot.py:28]: Discord.py v1.5.0

File: bot.py [hunk 2]
```diff
@@ -25,6 +25,9 @@
 from utils.caches import cache, CacheManager
 from logging.handlers import RotatingFileHandler
 
+# this section is for the new gateway (latest discord.py version)
+intents = discord.Intents.all()
```
Comment (CnyAnime) [inline bot.py:29]: Discord.py v1.5.0

**Comment by TheMoksej** [comment]
Thanks, I'll check that out later today.

**Comment by CnyAnime** [comment]
> Thanks, I'll check that out later today.


Ok

**Review by TheMoksej — APPROVED** [review]
Thanks!

**Pull request merged**